c7n/tags.py

```diff
@@ -80,8 +80,16 @@ def universal_augment(self, resources):
     if not resources:
         return resources
 
+    # For global resources, tags don't populate in the get_resources call
+    # unless the call is being made to us-east-1
+    if hasattr(self.resource_type, 'global_resource') \
+       and self.resource_type.global_resource:
+        region = 'us-east-1'
+    else:
+        region = self.region
+
     client = utils.local_session(
-        self.session_factory).client('resourcegroupstaggingapi')
+        self.session_factory).client('resourcegroupstaggingapi', region)
```

Comment:
can you pass region in explicitly via keyword mapping here `region_name=region`.